source/includes/edit.md

```diff
@@ -0,0 +1,246 @@
+        "seller_gstin": "1234",
+        "taxable_value": 100,
+        "ewaybill_serial_number": "2345677",
+        "is_seller_registered_under_gst": false,
+        "sgst_tax_rate": 100,
+        "place_of_supply": "DELHI",
+        "gst_discount": 0,
+        "hsn_code": "1234",
+        "sgst_amount": 100,
+        "enterprise_gstin": "13",
+        "gst_total_tax": 100,
+        "igst_amount": 100,
+        "cgst_amount": 200,
+        "gst_tax_base": 200,
+        "consignee_gstin": "1233",
+        "igst_tax_rate": 100,
+        "invoice_reference": "1234",
+        "cgst_tax_rate": 100
+    },
+    "additional": {
+        "rvp_reason": "Customer wants to return the product",
+        "async": false,
+        "label": true,
+        "order_date": "2020-01-30T20:14:34",
+        "vendor_code": "ADDR12182_1463",
+        "order_id": "Order Number of the Shipment",
+        "invoice_base_64": "base_64_string_of_invoice",
+        "duty_fee_paid_by":"S"
+    },
+    "awb_number":"111"
+}
+
+```
+
+>__Response__
+
+```json
+{
+    "meta": {
+        "status": 200,
+        "message": "Order edited successfully",
+        "success": true
+    }
+}
+```
+
+It’s a POST request as follows
+
+URL:
+`https://www.clickpost.in/api/v1/edit-order/`
+
+Headers: {'Content-type': 'application/json'}
+
+####URL Parameters:
+
+Parameter | Type | Description
+--------- | ---- | -----------
+key | character | API key provided to you
+
+####POST Parameters:
+
+Format: JSON
+
+####Compulsory Fields
+
+Payload is a list of json objects each of which have following fields:
+
+Parameter | Type | Description
+--------- | ---- | -----------
+awb | character | awb number
+
+####Optional Fields
+
+In case you want the recommendation logic to incorporate pricing as a parameter as well please pass the following fields in the API. If not passed, the pricing given by you in the system will not be entertained as a parameter.
+
+Parameter | Type | Description
+--------- | ---- | -----------
+weight | integer | weight of the shipment
+length | integer | length of the shipment
+breadth | integer | breadth of the shipment
+height | integer | height of the shipment
+pickup_info | object | sender information
+drop_info | object | consignee information
+additional | object | additional information
+shipment_details | object | item details
+gst_info | object | gst information
+
+####Response explaination:
+
+Response object has two parts:
+
+1. meta: stores information about the API, success or failure
+    + success: true/false, tells whether the order was created or not
+    + message: SUCCESS in case order was edited successfully, else returns error message.
+    + status:
+        1. 200 if the order is created successfully,
+        2. 400 if there is a bad request encountered: errors will be present in “message”
+
+
+<aside class="warning">
+You must replace key with key provided to you.
+</aside>
+
+####Note
+
+This api is currently being used by 3 clients only.
```

Comment:
This API is applicable for 3 courier partners [not clients]

Reply:
Can we also add fields that each of the courier partner allows editing? Will be mentioned in their API docs

Reply:
yes already added after generalized structure.